server/users.js

```diff
@@ -606,8 +609,9 @@ class User extends Chat.MessageContext {
 	 */
 	getIdentityWithStatus(roomid = '') {
 		const identity = this.getIdentity(roomid);
-		if (!this.status) return identity;
-		return `${identity}@${this.status}`;
+		const statusSymbol = this.statusType === 'busy' ? '!' : this.statusType === 'idle' ? '?' : '';
+		const status = statusSymbol + (this.userMessage || '');
```

Comment:
When can `this.userMessage` be a false-y value?

Reply:
I suppose it can only be ''. Decided to do this just to be safe. I guess it can be removed.

Reply:
I like the safety here.